webpages/api/brainstorm/load_brainstorm.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
-if (!include ('../../../db_name.php')) {
-    include ('../../../db_name.php');
+if (!include ('../../config/db_name.php')) {
+    include ('../../config/db_name.php');
```

Comment:
Does the second include do anything? I think the purpose of a structure like this is if the first include doesn't find the file, it returns false, and then the second one can try an alternative location. Not sure a second include of the same location is useful.

Reply:
Huh. I *thought* I just copied a heuristic from elsewhere in the code... but the piece of code that I believed that I copied is in db_functions and it uses a file_exists followed by a require_once. 

So, I think when I first set that up, I was just a PHP n00b, and I've been copying and pasting it since.

Reply:
Addressed in f774f3e4e9edd65861e9caee3e47fab6464f139e